manifests/server/conf.pp

```diff
@@ -331,6 +338,39 @@
     }
   }
 
+  # By default, remove weak ciphers and allow users to set a minimum TLS
+  # protocol if openlap supports it
+  if $tls_cipher_suite {
+    $_tls_cipher_suite = $tls_cipher_suite
+  }
+  if $facts['slapd_version'] {
+    if $facts['slapd_version'] >= '2.4.40' {
+      if $tls_protocol_min {
+        $_tls_protocol_min = $tls_protocol_min
+      }
+      # Minimum bound of TLSv1.2
+      else {
+        $_tls_protocol_min = '3.3'
+      }
+      # If the minimum TLS bound is > TLSv1.0, remove weak protocols
+      if !defined('$_tls_cipher_suite') and $_tls_protocol_min >= '3.1' {
+        $_tls_cipher_suite = ['HIGH','-TLSv1.0', '-SSLv3', '-SSLv2']
```

Comment:
I scraped sslv2 from the man page, but it's so out of date openssl does not even provide it (good thing, too...). I'll remove it.